subprojects/core/src/integTest/groovy/org/gradle/cache/internal/DefaultGeneratedGradleJarCacheIntegrationTest.groovy

```diff
@@ -49,6 +50,11 @@ class DefaultGeneratedGradleJarCacheIntegrationTest extends Specification {
 
     def DefaultServiceRegistry services = (DefaultServiceRegistry) ServiceRegistryBuilder.builder()
             .parent(NativeServicesTestFixture.getInstance())
+            .provider(new Object() {
```

Comment:
Maybe we should have something like a NO_OP logging service registry that we can wire in these cases? That would avoid some repetition.

Reply:
Good idea! Done in 3b4b7114c36.